crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -916,7 +916,7 @@ New experimental version of nrf_cc310_mbedcrypto with changes to platform initia
 
 ..warning::
    This version may lead to undefined behavior on some nRF9160 devices.
-   Please use a newer version.
+   Hence, use a newer version.
```

Comment:
Change the warning to a note. You remember, no warnings unless someone's health or life is at risk.